docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,136 @@
+
+    ```yaml
+    apiRules:
+      - name: incoming-http-trigger
+        service:
+          host: incoming
+        rules:
+          - methods:
+              - GET
+            accessStrategies:
+              - handler: allow
+    ```
+
+3. Provide your Function logic in the `handler.js` file:
+>**NOTE:** In this example, there's no sanitization logic. The `sanitize` Function is just a placeholder.
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports = {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+   >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names) page. The receiver subscribes to the event type to consume the events.
+
+   >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
+
+4. Apply your emitter Function:
+
+    ```bash
+    kyma apply function
+    ```
+   Your Function is now built and deployed in Kyma runtime. Kyma exposes it through the API Rule. The incoming payloads are processed by your emitter Function. It then sends the sanitized content to the workload that subscribes to the selected event type. In our case, it's the receiver Function.
+
+5. Test the first Function. Send the payload and see if your HTTP traffic is accepted:
+
+      ```bash
+      export KYMA_DOMAIN={KYMA_DOMAIN_VARIABLE}
+      
+      curl -X POST https://incoming.${KYMA_DOMAIN}
+      -H 'Content-Type: application/json'
+      -d '{"foo":"bar"}'
+      ```
+### Create the receiver Function
+
+1. Go to your `receiver` folder and run Kyma CLI `init` Kyma CLI command to initialize the scaffold for your second Function:
+   ```bash
+   kyma init function
+   ```
+2.  The `init` command creates the same files as in the `emitter` folder.
```

Comment:
We can make it a regular sentence under the command, not necessarily another step 🤔 Especially that you made it a regular sentence in the previous paragraph.